pkg/container/container.go

```diff
@@ -62,6 +71,21 @@ type Container interface {
 	// SelinuxLabel returns the container's SelinuxLabel
 	// it takes the sandbox's label, which it falls back upon
 	SelinuxLabel(string) ([]string, error)
+
+	// spec functions
+
+	// returns the spec
+	GetSpec() *generate.Generator
+
+	// AddMountToSpec adds a mount to the container's spec
+	// it takes the destination, the mount type, the source type, the mount to remove, and the options
+	AddMountToSpec(string, string, string, string, []string)
```

Comment:
maybe `AddSpecMount` or below have it be `AddAnnotationsToSpec`. Whichever one, but I'd like them to be of the same structure

Reply:
maybe it should be SpecAddMount, and have AddAnnotationsToSpec be SpecAddAnnotations ? so when you're trying to use them they would be grouped by "spec" and be easy to use if you're using autocomplete ?

Reply:
that's an option! another one that I just thought of is just do `AddAnnotations` and `AddMounts` and do away with specifying the spec

Reply:
I am still not too opinionated as long as they're consistent :smiley:

Reply:
for sure lol

Reply:
renamed!